hudi-io/src/main/java/org/apache/hudi/storage/StorageSchemes.java

```diff
@@ -110,6 +115,10 @@ public boolean isAtomicCreationSupported() {
     return supportAtomicCreation != null && supportAtomicCreation;
   }
 
+  public boolean getListStatusFriendly() {
+    return listStatusFriendly != null && listStatusFriendly;
```

Comment:
Can we enumerate the listing friendly scheme types instead of extending the enum fields, it looks like only `FSUtils.getPathInfoUnderPartition` use this method. we can extend it in the future if more use cases need it.